src/utils/hash.ts

```diff
@@ -17,3 +21,13 @@ export async function hashStream(stream: Readable): Promise<string> {
     stream.on('error', reject);
   });
 }
+
+/**
+ * Create a multihash from a string or Buffer
+ *
+ * @param x a string or `Buffer`
+ * @returns sha256 multihash string of the string or `Buffer`
+ */
+export function hashString(x: Buffer | string): string {
```

Comment:
It looks like every production use of this takes a `Buffer`, so we should probably do the same. Alternatively, we could copy the [type of the `hash.update` `data` parameter](https://nodejs.org/api/crypto.html#hashupdatedata-inputencoding).

Also, the name `x` for the input should probably be something more readable. This might be one of the rare times when `data` is actually appropriate.